libraries/SITL/examples/test/plane_formation.sh

```diff
@@ -0,0 +1,47 @@
+#Author:Shreya Kulsh
+#This program tests a formation between three planes
+#!/bin/bash
+
+# three plane swarm (x offset formation)
```

Comment:
move the plane into the follow directory, and rename to plane_3_formation or something like that